service/src/main/java/io/camunda/service/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -72,13 +72,12 @@ public SearchQuery toSearchQuery(final UserTaskFilter filter) {
 
   @Override
   public List<String> toIndices(final UserTaskFilter filter) {
-    final var completed = filter.completed();
-    final var canceled = filter.canceled();
+    final var created = filter.created();
 
-    if (completed || canceled) {
-      return Arrays.asList("tasklist-task-8.5.0_alias");
-    } else {
+    if (created) {
```

Comment:
if the filter has `created == true` and `completed == true`, we need to use the alias.
But I said above, it is not clear when we use these booleans and when to use the `states` list in UserTasklistFilter

Reply:
The boolean is just a more direct way to apply the filters (my inspiration was based on ProcessInstanceFilter implementation)

Reply:
True, this is redundant and has better way to work on this